src/lib/engine/organization.js

```diff
@@ -36,3 +36,5 @@ export function organizationMemberships(organizationId, memberships) {
 export function organizationInvitations(organizationId, invitations) {
   return _.filter(invitations, e => sameId(e.organization_id, organizationId))
 }
+
+export const getFactVar = ({id}) => `organization_${id}`
```

Comment:
I'm not a big fan of this name.  Maybe, ``organization_readable_id`` or similar?